examples/package.json

```diff
@@ -2,12 +2,12 @@
   "name": "@solarwinds-apm/examples",
   "private": true,
   "scripts": {
-    "lint": "prettier --check . && eslint --max-warnings=0 *.js '*/**/*.{js,ts}'",
-    "lint:fix": "eslint --fix *.js '*/**/*.{js,ts}' && prettier --write ."
```

Comment:
Realised eslint got smarter and I don't have to specify the files to lint manually anymore :D